### PR TITLE
Add Slim support

### PR DESCRIPTION
**Pull request by veganstraightedge**
This is a work-in-progress PR to add Slim support.

Since I'm not able to get all of the [tests to run](https://github.com/robotdana/leftovers/issues/10#issuecomment-1033341458), at the moment, this was an attempt to follow [your guidance](https://github.com/robotdana/leftovers/issues/10#issuecomment-1014894456) copying haml files/methods/configs/etc into slim ones.

### CODE REVIEW

**Comment by robotdana** [comment]
looks good so far :)

**Comment by veganstraightedge** [comment]
I've gotten down to five failures. Four of which are the same failure. I don't think I'm actually calling Slim's Engine correctly/completely. But sadly, I don't understand the either Slim or leftovers well enough to understand what I don't understand. 😐

If you have some bandwidth sometime, could you take a look and see if you see what I don't see, please?

File: leftovers.gemspec [hunk 1]
```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| ::File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'activesupport'
+  spec.add_development_dependency 'activesupport', '< 7'
```
Comment (robotdana) [inline leftovers.gemspec:33]: reminder to myself to not merge while this is still here

**Comment by robotdana** [comment]
I'm happy to merge this if you're happy with my changes, with dropping out the temp commit (which i'll leave for you to do so i'm not surprise force pushing)

thanks so much for your contribution :)

**Comment by veganstraightedge** [comment]
If you're happy with it, I'm happy with it. 🎉
Thanks for guiding me through it. 

:shipit:

**Comment by veganstraightedge** [comment]
And whenever you cut a new `0.6.0` release, I'll try it out on our large production Rails apps at work, to see what we see.

**Comment by robotdana** [comment]
it's released as v0.5.4 :)

**Pull request merged**